src/www/ui/api/documentation/openapi.yaml

```
@@ -3981,6 +4006,40 @@ components:
         isCandidate:
           description: Is the license a candidate?
           type: boolean
+    Customise:
+      description: Admin customise ddata object
+      type: object
+      properties:
+        key:
+          description: field key
+          type: string          
+        value:
+          description: field value
+          type: string
+        type:
+          description: field type
+          type: string
+          enum:
+          - int
+          - text
+          - textarea
+          - password
+          - dropdown
+          - boolean
+
+        label:
+          description: field label
+          type: string
+        group_name:
+          description: group name for the field
+          type: string
+        group_order:
+          description: group order of the field
+          type: integer
+        option_value:
+              description: If type is 'dropdown', provide options in format op1{val1}|op2{val2}|...
+              type: string
+            
```

Small formatting changes needed.
```suggestion
          - boolean
        label:
          description: field label
          type: string
        group_name:
          description: group name for the field
          type: string
        group_order:
          description: group order of the field
          type: integer
        option_value:
          description: If type is 'dropdown', provide options in format op1{val1}|op2{val2}|...
          type: string

```